components/map/map.service.ts

```diff
@@ -505,6 +500,45 @@ export class HsMapService {
       });
   }
 
+  /**
+   * @param lyr {Layer} Layer which to proxify if needed
+   * @description Proxify layer based on its source object type and if its tiled or not.
+   * Each underlying OL source class has its own way to override imagery loading.
+   */
+  proxifyLayer(lyr: Layer): void {
+    const source = lyr.getSource();
+    if (
+      [ImageWMS, ImageArcGISRest].some((typ) =>
+        this.HsUtilsService.instOf(source, typ)
+      )
```

Comment:
```suggestion
      [ImageWMS, ImageArcGISRest].some((type) =>
        this.HsUtilsService.instOf(source, type)
      )
```

Reply:
I was using it on purpose not to conflict with the type keyword.

Reply:
Oh, I tought its just typo ....well but its just variable in a specific context, dont think it could be problematic in any way. However for the same reason I might also dont mind 'typ' then...it just kinda feels bit strange.